apps/avifenc.c

```diff
@@ -687,11 +687,11 @@ static avifBool readEntireFile(const char * filename, avifRWData * raw)
     return AVIF_TRUE;
 }
 
-// Returns NULL if a memory allocation failed.
+// Returns NULL if a memory allocation failed. The return value should be freed with free().
 static char * avifStrdup(const char * str)
 {
     size_t len = strlen(str);
-    char * dup = avifAlloc(len + 1);
+    char * dup = malloc(len + 1);
```

Comment:
entry->value uses that below and is freed through avifFree. So it should be avifAlloc-ed right?

Reply:
avifFree() was not used in this file before this pull request. The return values of avifStrdup() are freed by free() here:

```
static void avifCodecSpecificOptionsFree(avifCodecSpecificOptions * options)
{
    while (options->count) {
        --options->count;
        free(options->keys[options->count]);
        free(options->values[options->count]);
    }
    ...
}
```